.gitignore

```diff
@@ -65,3 +65,4 @@ target/
 pyuvdata/data/
 pyuvdata/data/test/
 pyuvdata/GIT_INFO
+uvdata/data/test/*
```

Comment:
That's strange. The location that test data is written is usually pyuvdata/data/test. If it's getting written somewhere else we should understand why.